test/launchdarkly/mzcompose.py

```diff
@@ -84,7 +84,7 @@ def workflow_default(c: Composition) -> None:
         # Assert that the default max_result_size is served when sync is disabled.
         with c.override(Materialized()):
             c.up("materialized")
-            c.testdrive("\n".join(["> SHOW max_result_size", "1073741824"]))
```

Comment:
@nrainer-materialize I updated the failing launchdarkly test. This was failing since now the value is displayed with a unit.

Reply:
Thank you!